packages/next-server/lib/fetch/follow.ts

```diff
@@ -0,0 +1,40 @@
+import { parse } from 'url'
```

Comment:
I would change this because `url` is going to be deprecated https://nodejs.org/docs/latest/api/url.html#url_legacy_url_api

Reply:
I'll update, thank you 👍